python-sdk/tests/test_sdk.py

```diff
@@ -116,8 +147,15 @@ def test_detect_injection_heuristics(
         assert not rebuff_response.injection_detected
 
 
+@pytest.mark.parametrize(
+    "rebuff",
+    [VectorDB.PINECONE, VectorDB.CHROMA],
+    ids=["pinecone", "chroma"],
+    indirect=True,
+)
 def test_detect_injection_vectorbase(
     rebuff: RebuffSdk,
+    add_documents_to_chroma,
```

Comment:
This parameter isn't getting used in this method, it probably shouldn't be a parameter. What I would probably do is add `add_documents_to_chroma` to the end of the `rebuff` function fixture.